python/pacemaker/_cts/CTS.py

```diff
@@ -178,8 +202,24 @@ def wait_for_all_nodes(self, nodes, timeout=300):
 
 
 class Process:
+    """ A class for managing a process """
```

Comment:
FYI "process" here refers to a Pacemaker daemon

Reply:
Fixed